huawei_solar_battery_optimization.yaml

```diff
@@ -966,7 +983,7 @@ automation:
                         state: 'off'
                       - condition: template
                         value_template: >
-                          {{ states(inverter_entity) != "Limited to 100.0%" }}
+                          {{ states(inverter_entity) != "Limited to 100.0%" }} #Forslag: Blot kald det for "Reset Export"
```

Comment:
parameteren der sættes er "Limited to 100.0%" - så kan ikke ændre denne tekst.... men hvis vi alligevel ændrer væk fra % og over på W så skal denne alligevel ændres.